requirements/constraints.txt

```diff
@@ -8,12 +8,18 @@
 # pin when possible.  Writing an issue against the offending project and
 # linking to it here is good.
 
-# Common constraints for edx repos
--c https://raw.githubusercontent.com/edx/edx-lint/master/edx_lint/files/common_constraints.txt
-
 # jsonfield2 > 3.0.3 dropped support for python 3.5
 jsonfield2==3.0.3
 
 # pinning it to latest release.
 celery==5.0.4
 
+# docutils version 0.17 is causing docs rendering to fail
```

Comment:
If we use this approach then we should also add a note in the common_constraints file to update this repo separately if there is any future common constraint added in it.